.tekton/tasks/check-pr-rebase-label.yaml

```diff
@@ -0,0 +1,70 @@
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+  name: pr-rebase-labels
+spec:
+  description: |
+    check the pull request label for which the test is running on staging is needs rebase or not.
+  params:
+    - name: match_label
+    - name: pull_request_number
+    - name: TARGET_GH_OWNER
+      description: Owner of GitHub repository of the infra-deployments code
+      default: redhat-appstudio
+    - name: TARGET_GH_REPO
+      description: GitHub repository of the infra-deployments code
+      default: infra-deployments
+  results:
+    - name: matched-rebase-label
+  workspaces:
+    - name: shared-file-path
+  steps:
+    - name: check-pr-rebase-label
+      image: quay.io/redhat-appstudio/github-app-token@sha256:b4f2af12e9beea68055995ccdbdb86cfe1be97688c618117e5da2243dc1da18e
+      workingDir: $(workspaces.shared-file-path.path)
+      env:
+        - name: GITHUB_API_URL
+          value: https://api.github.com
+        - name: PULL_REQUEST_NUMBER
+          value: $(params.pull_request_number)
+        - name: TARGET_GH_OWNER
+          value: "$(params.TARGET_GH_OWNER)"
+        - name: TARGET_GH_REPO
+          value: "$(params.TARGET_GH_REPO)"
+      script: |
+        #!/usr/bin/env python3
```

Comment:
Why check for the label instead of checking if it needs to be rebased?

I was expecting a command such as:
```
if [ "$(git log HEAD ..main) | wc -l" == "0" ]; then
  echo "False"
else
  echo "True"
fi
```